static/js/protein/requests.ts

```diff
@@ -0,0 +1,68 @@
+import axios from "axios";
+
+import {
+  deduplicateInteractionDCIDs,
+  V1BaseDatum,
+  V1Response,
+  valuesFromResponse,
+} from "./data_processing_utils";
+
+const V1_ENDPOINT_ROOT = "https://autopush.api.datacommons.org/v1";
```

Comment:
Oh oops thanks for catching! I think flask so I'll do `/api/...`.

Reply:
looks like you're directly using mixer api's (there are no code changes from your end in flask? thats python running in the website component..). did you run into any cors issues when you did this? we use passthroughs for api's in flask for al our calls.

ideally, this could be translated to a single request from react to flask, and flask does the bulk request, data munging etc and returns the cleaned data to the client for easy rendering.

Reply:
Autopush might run into cors issues even if your local isn't running into them. Please create flask endpoint(s) for retrieving the data.

To create the flask endpoint:

- in datacommons.py, add the code at line 156-168 in the datacommons.py file from this PR: https://github.com/shifucun/website/commit/79046d4923e7fd1b027cfcf0c24ea92b5249cdc6#diff-be4c9d19b450d78184e8477cd2184bbf978a95d94cf68c359cd08deb7940ff63
- In server/routes/api/protein.py you can add your flask endpoints. (example of a flask POST endpoint: https://github.com/datacommonsorg/website/blob/master/server/routes/api/stats.py#L286-L293)

Reply:
Oops sorry I missed this! Thanks for the detailed instructions I'll get going on this!

Reply:
Thanks this was very helpful! So I was able to make a new flask endpoint in `server/routes/api/protein.py` but the `property_values` function didn't directly work (I got a dict not hashable error).  I'm sure it was some small thing but I didn't know what the original post response format was so I didn't attempt to match the old output.  So then instead I just made a `get_response` function which makes a post request and then forwards the response to flask, and my flask endpoint currently just forwards this response again to the client.  What do you think?

Reply:
(Discussed this in GVC with @chejennifer, plan is to still use `property_values` after consulting @shifucun about the intended spec.)